src/backend/LDW.WebAPI/Startup.cs

```diff
@@ -24,6 +25,8 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
 
+            services.AddMvc(options => { options.Filters.Add<GlobalExceptionFilter>(); });
+
             #region Swagger
```

Comment:
Swagger? So will we have our own Swagger API page like [this](https://petstore.swagger.io/)?

Reply:
Yes:) @sergiynikon already added swagger to api